<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="7F6808" failures="1" tests="1" name="Bai26Parallel.Testcase.TestcaseSignin" time="18.149" errors="0" timestamp="2022-12-10T15:06:17 CST" skipped="0">
  <testcase classname="Bai26Parallel.Testcase.TestcaseSignin" name="LoginFailed1" time="18.149">
    <failure message="hic expected [-1000002
user not allowed to log in345.] but found [-1000002
user not allowed to log in.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: hic expected [-1000002
user not allowed to log in345.] but found [-1000002
user not allowed to log in.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at salary.com.utils.WebUI.verifyEquals(WebUI.java:68)
at Bai26Parallel.Pages.PageSignin.Loginwrongusername(PageSignin.java:46)
at Bai26Parallel.Testcase.TestcaseSignin.LoginFailed1(TestcaseSignin.java:87)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- LoginFailed1 -->
  <system-out/>
</testsuite> <!-- Bai26Parallel.Testcase.TestcaseSignin -->
